mevolib/fetch/BioSeqs.py

```diff
@@ -367,174 +304,151 @@ def from_entrez (cls, email, entrez_db, query, max_fetch = sys.maxsize):
         report = [(date_time, 'entrez', entrez_db, query)]
         return (cls(seq_dict, report))
 
-    def __len__ (self):
-        """
-        Returns:
-            int
-                Total number of sequences.
+
+    def __len__ (self) -> int:
+        """Total number of sequences.
```

Comment:
```suggestion
        """Returns the total number of sequences."""
```
And remove next `"""`.